Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -51,10 +51,27 @@
             margin-left: auto;
             margin-right: auto;
             margin-top: -40px;
-            /* Small Devices*/
-            @media (max-width: 767px) {
+
+            @media screen and (max-width: 575px) {
```

Comment:
these @media styles tend to be tricky to debug. On top of that bootstrap handles the media styles by itself when using lg/md/sm . We should use them sparingly.

Reply:
not sure if @media is not recommended though and not sure why debugging is an issue here as you can always update your css for screen to be refreshed right? I think it is indeed a great approach which is even officially documented and recommended on the bootstrap's website <https://getbootstrap.com/docs/4.1/layout/overview/#responsive-breakpoints> and we need to be familiar with it.

Reply:
The whole point of having bootstrap handle different sizes is for us to not mess with custom css for different media sizes. That is why is not recommended.